ui/decredmaterial/card.go

```diff
@@ -11,32 +11,59 @@ import (
 
 type Card struct {
 	layout.Inset
-	Color   color.RGBA
-	Rounded bool
+	Color       color.RGBA
+	Rounded     bool
+	CornerStyle CornerStyle
 }
 
+type CornerStyle uint8
+
+const (
+	SquareEdge CornerStyle = iota
+	HalfRoundedEdgeTop
+	HalfRoundedEdgeBottom
+	RoundedEdge
+)
+
```

Comment:
rather than having defined radius shapes for the card widget, why not create a struct type "CardRadius" where the radius of each corner of the card can be specified. It makes the card widget more flexible, the constant cardRadius value can instead be used as a default value when a CardRadius data type isn't passed to the widget.